src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -1853,6 +1853,23 @@ public void ConnectCallback_GetSet_Roundtrips()
             }
         }
 
+        [Fact]
+        public void PlaintextStreamFilter_GetSet_Roundtrips()
+        {
+            using (var handler = new SocketsHttpHandler())
+            {
+                Assert.Null(handler.PlaintextStreamFilter);
+
+                Func<SocketsHttpPlaintextStreamFilterContext, CancellationToken, ValueTask<Stream>> f = (context, token) => default;
+
+                handler.PlaintextStreamFilter = f;
+                Assert.Equal(f, handler.PlaintextStreamFilter);
```

Comment:
Nit:
```suggestion
                Assert.Same(f, handler.PlaintextStreamFilter);
```
but don't restart CI just for this